xml/System.Runtime.ExceptionServices/ExceptionDispatchInfo.xml

```diff
@@ -126,10 +126,9 @@
         <Parameter Name="source" Type="System.Exception" Index="0" FrameworkAlternate="net-5.0" />
       </Parameters>
       <Docs>
-        <param name="source">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="source">The unthrown <see cref="T:System.Exception"/> instance.</param>
```

Comment:
```suggestion
        <param name="source">The unthrown exception.</param>
```

Parameter and return descriptions should avoid mentioning the data type. See https://github.com/dotnet/dotnet-api-docs/wiki/Parameters%3A-Methods-and-constructors. The data type is shown automatically above the description.